src/main/java/com/brightcove/castlabs/client/CastlabsClient.java

```diff
@@ -24,14 +24,15 @@
 
 import com.brightcove.castlabs.client.request.IngestKeysRequest;
 import com.brightcove.castlabs.client.response.IngestAssetsResponse;
+import com.brightcove.castlabs.client.request.AddSubMerchantAccountRequest;
+import com.brightcove.castlabs.client.response.AddSubMerchantAccountResponse;
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 /**
  * Client for interacting with the Castlabs key ingestion API.
- * 
- * @author Scott Kidder
  *
+ * @author Scott Kidder
```

Comment:
Is it necessary to maintain static reference to what git would otherwise provide us?